OmiseSwiftTests/ChargesOperationFixtureTests.swift

```diff
@@ -1137,7 +1137,54 @@ class ChargesOperationFixtureTests: FixtureTestCase {
         XCTAssertEqual(items[4].name, "metadata[customer_id]")
         XCTAssertEqual(items[4].value, "123")
     }
+
+    func testMobileBankingChargeCreate() {
```

Comment:
I think we need a more specific `Bank` on the method name too, like `testSCBMobileBankingChargeCreate`.